erpnext/manufacturing/doctype/job_card/job_card.json

```diff
@@ -280,7 +280,7 @@
    "read_only": 1
   },
   {
-   "fetch_from": "production_item.item_name",
+   "fetch_from": "work_order.production_item.item_name",
```

Comment:
please respond to this first https://github.com/frappe/erpnext/pull/26386#discussion_r685045376